### PR TITLE
docs: long lived testnets docs

### PR DESCRIPTION
**Pull request by auricom**

### CODE REVIEW

File: long-lived-testnets/v2-052-1/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+# `v2-052-1` Chain Details
+
+* **Chain-ID**: `v2-052-1`
+* **denom**: `stake`
+* **Current Cosmos SDK upstream**: [`v0.52.x`](https://github.com/cosmos/cosmos-sdk/tree/release/v0.52.x)
```
Comment (julienrbrt) [inline long-lived-testnets/v2-052-1/README.md:6]: Can we directly link the simapp of the branch in those docs? it is easy to see what is being built directly.

File: long-lived-testnets/052-1/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+# `052-1` Chain Details
+
+* **Chain-ID**: `052-1`
+* **denom**: `stake`
+* **Current Cosmos SDK upstream**: [`v0.52.x`](https://github.com/cosmos/cosmos-sdk/tree/release/v0.52.x)
```
Comment (julienrbrt) [inline long-lived-testnets/052-1/README.md:6]: ditto

File: long-lived-testnets/050-1/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+
+# `050-1` Chain Details
+
+* **Chain-ID**: `050-1`
+* **denom**: `stake`
+* **Current Cosmos SDK upstream**: [`v0.50.x`](https://github.com/cosmos/cosmos-sdk/tree/release/v0.50.x)
```
Comment (julienrbrt) [inline long-lived-testnets/050-1/README.md:6]: ditto

**Review by julienrbrt — APPROVED** [review]
nice! just one nit

**Review by tac0turtle — APPROVED** [review]

File: long-lived-testnets/v2-052-1/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+
+# `v2-052-1` Chain Details
+
+* **Chain-ID**: `v2-052-1`
+* **denom**: `stake`
+* **Application**: [`Simapp - Cosmos-SDK v0.50.x`](https://github.com/cosmos/cosmos-sdk/tree/release/v0.50.x/simapp)
```
Comment (julienrbrt) [inline long-lived-testnets/v2-052-1/README.md:6]: copy/pasty mistake, should be pointing to simapp v2 in release/v0.52

Comment (auricom) [reply long-lived-testnets/v2-052-1/README.md:6]: fixed, thanks !

**Pull request merged**